### PR TITLE
Nagant revolver

### PR DESCRIPTION
**Pull request by SonixApache**
A complaint was that 'Matebas are just reskinned traitor revolvers and you can get them super cheap and easy from the goonsat" so now one part of that complaint is fixed.
nagantrevolver
Holds 7 shells, fires bullets that do 25 brute, can be silenced. 
Ammo is unique to the gun, good luck finding any.

balance: **russians now fire a way weaker projectile; 60 -> 25**

Internally changes the Mosin Nagant's name from `nagant` to `mosin` to help differentiate.

:cl:
- tweak: russians now get nagant revolvers which are way weaker than matebas

### CODE REVIEW

**Comment by boy2mantwicethefam** [comment]
Hmm
Hmm

**Comment by Blithering** [comment]
Don't forget the recent AEG sprite update happened
Also it was bugging me they had matebas, glad I'm not the only autist in these matters. now we just need someone to port russian equipment over to deity's nazi mobs so they can have all that fancy weapon switching AI

**Comment by SonixApache** [comment]
i swear to god if this PR gets murdered by fucking icon conflicts again

**Comment by boy2mantwicethefam** [comment]
Serves you right for making snowflaked ammo that you can't get anywhere

**Comment by SonixApache** [comment]
i can just add it to the russian weapons crate i guess

**Comment by Blithering** [comment]
I'd say put it in the goonecode ammo dump, but with the vaultification PR its gonna make conflicts.

**Comment by SonixApache** [comment]
yeah that makes more sense anyway, will do that after (or make robo add it if mine goes first)

File: __compile_options.dm [hunk 1]
```diff
@@ -25,25 +25,25 @@
...
 // If 1, the server stops after the tests are done
 #define UNIT_TESTS_STOP_SERVER_WHEN_DONE 0
 
 #if DEVELOPER_MODE
 // If defined, overrides the default lobby timer duration
-#define GAMETICKER_LOBBY_DURATION 5 SECONDS
+#define GAMETICKER_LOBBY_DURATION 7 SECONDS
 #endif
 // If 1, mob/Login checks for multiple connections from the same IP on different ckeys and warns the user
 #define WARN_FOR_CLIENTS_SHARING_IP !DEVELOPER_MODE
```
Comment (SonixApache) [inline __compile_options.dm]: GIT, STOP

**Comment by Blithering** [comment]
its evolving!

**Comment by juanototo2** [comment]
why? who or what hurt you?

**Comment by Dacendeth** [comment]
Does this mean the russain mobs will fire less dangerous bullets as well?

**Comment by SonixApache** [comment]
yes

**Comment by FudgePucker** [comment]
So who's going to make the Russian traitor bundle?

**Pull request merged**